### DISCUSSION TITLE
Does CLEAR CACHE options effect current cube or all cubes?

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [DaxStudio/DaxStudio#1321] [Q&A]
Does CLEAR CACHE options effect current cube or all cubes?
It seems dangerous to clear cache testing code in single model if it can affect all SSAS models.
Is there a tool that can definitively show what is happening?

**Comment by dgosbell** [comment]
As documented https://daxstudio.org/docs/features/clear-cache/ it only affects the current database (and most databases now only have a single cube/model). I use the [ClearCache XMLA command](https://learn.microsoft.com/en-us/analysis-services/xmla/xml-elements-commands/clearcache-element-xmla?view=asallproducts-allversions) and pass in the ID for the current database, which only affects the current "cube". There is no way to do a server wide clear cache.
